public/pages/Dashboard/Components/AnomaliesDistribution.tsx

```diff
@@ -170,67 +170,65 @@ export const AnomaliesDistributionChart = (
         <EuiFlexGroup justifyContent="center">
           <EuiFlexItem grow={false}>
             {isEmpty(anomalyDistribution) ? null : (
-              <Chart className="anomalies-distribution-sunburst">
-                <Partition
-                  id="Anomalies by index and detector"
-                  data={anomalyDistribution}
-                  valueAccessor={(d: Datum) => d.count as number}
-                  valueFormatter={(d: number) => d.toString()}
-                  layers={[
-                    {
-                      groupByRollup: (d: Datum) => d.indices,
-                      nodeLabel: (d: Datum) => {
-                        return d;
-                      },
-                      fillLabel: {
-                        textInvertible: true,
-                      },
-                      shape: {
-                        fillColor: (d) => {
-                          return fillOutColors(
-                            d,
-                            (d.x0 + d.x1) / 2 / (2 * Math.PI),
-                            []
-                          );
+              <div className="anomalies-distribution-sunburst">
```

Comment:
This is the only change to fix the issue: moving `sunburst` class to parent class. No changes to chart related code below